source/aggregation-pipeline-builder.txt

```diff
@@ -31,6 +31,15 @@ aggregation pipeline. The :guilabel:`Preview of Documents in the
 Collection` section shows 20 documents randomly sampled from the chosen
 collection.
 
+.. note:: Atlas Search Stages
+
+   When you connect |compass-short| to a MongoDB deployment hosted on 
+   `Atlas <https://www.mongodb.com/cloud/atlas?tck=docs_compass>`_,
+   additional Atlas-only stages 
+   :atlas:`$search </reference/atlas-search/query-syntax/#-search>` and
+   :atlas:`$searchMeta </reference/atlas-search/query-syntax/#-searchmeta>`
+   are available in the Aggregation Pipeline Builder.
```

Comment:
[suggestion]

Maybe add a sentence explaining when you might use these stages:

"Use these stages to perform full-text search on Atlas collections."

or something